Mathlib/Combinatorics/SimpleGraph/Finite.lean

```diff
@@ -73,10 +73,9 @@ theorem edgeFinset_ssubset_edgeFinset : G₁.edgeFinset ⊂ G₂.edgeFinset ↔
 
 @[gcongr] alias ⟨_, edgeFinset_mono⟩ := edgeFinset_subset_edgeFinset
```

Comment:
```suggestion
@[mono, gcongr] alias ⟨_, edgeFinset_mono⟩ := edgeFinset_subset_edgeFinset
```
I don't know that anyone really uses mono, but currently you are removing this tag: I think we should leave it for now.